assets/vue/components/BlockedIncident.vue

```diff
@@ -25,6 +25,7 @@
 </template>
 
 <script>
+import * as Filtering from '../helpers/filtering.js';
```

Comment:
I'm actually not sure what the best practice for naming here is (`Filtering`), but i tend towards lowercase. Uppercase is usually reserved for importing classes.

Reply:
I couldn't find much about it. I changed it to lower case. I guess it isn't worth updating the other PR and we can just merge this one.

Reply:
By the way, the reason I tended to upper-case is because it would be in-line with official APIs like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math. The import would basically create a similar "namespace object".

Reply:
Counterexample would be node namespaces like `fs.*`. 🤷  https://nodejs.org/dist/latest-v20.x/docs/api/fs.html#synchronous-api